pkg/routing/adapter/splitter/adapter.go

```diff
@@ -165,10 +165,7 @@ func (h *Handler) split(path string, e *event.Event) []*event.Event {
 	}
 	for _, v := range val.Array() {
 		newCE := cloudevents.NewEvent()
-		if err := newCE.SetData(cloudevents.ApplicationJSON, v.Raw); err != nil {
-			h.logger.Errorw("Failed to set event data", zap.Error(err))
-			continue
-		}
+		newCE.SetData(cloudevents.ApplicationJSON, v.Raw)
```

Comment:
just to be picky because you are sometimes :) why did you fix this in this PR about github actions ?

Reply:
It's explained in the description.

Reply:
ok intentional fmt breaks